Fix the color slider with a recent GTK, to fix #467431 [1], patch from
upstream #19816 [2].

[1] https://bugzilla.redhat.com/show_bug.cgi?id=467431
[2] http://inkscape.svn.sourceforge.net/viewvc/inkscape/inkscape/trunk/src/widgets/sp-color-slider.cpp?r1=16358&r2=19816

--- inkscape/trunk/src/widgets/sp-color-slider.cpp	2007/10/30 06:16:09	16358
+++ inkscape/trunk/src/widgets/sp-color-slider.cpp	2008/09/08 07:52:36	19816
@@ -330,8 +330,11 @@
 	g_return_if_fail (SP_IS_COLOR_SLIDER (slider));
 
 	if (!adjustment) {
-		adjustment = (GtkAdjustment *) gtk_adjustment_new (0.0, 0.0, 1.0, 0.01, 0.1, 0.1);
-	}
+		adjustment = (GtkAdjustment *) gtk_adjustment_new (0.0, 0.0, 1.0, 0.01, 0.0, 0.0);
+	} else {
+        gtk_adjustment_set_page_increment(adjustment, 0.0);
+        gtk_adjustment_set_page_size(adjustment, 0.0);
+    }
 
 	if (slider->adjustment != adjustment) {
 		if (slider->adjustment) {
